### DISCUSSION TITLE
Could we have a UI Scale setting?    font/button/menu size adjustment

### DISCUSSION THREAD

**Discussion by subatomicglue** [rern/rAudio#604] [Fixed Issues] [answered]
I have a 7" touchscreen running RPI4b and 20211022 and I constantly have to squint to see the screen, and 20% of the time I can't fully read what's there.

For reference, I use a 16" monitor all day long and have no troubles with normal display size.

Additionally, the menu items under Settings are super hard to read/target, though surprisingly it does a good job guessing what my fat thumb is wanting when it covers up 3 at a time :-)   would feel better if those were larger menu items (scale with the UI scale feature I'm asking for).

An organizational request for Settings:

And man, I get such dyslexia between all the menu settings, I usually go through 4 of them to find the switch that I know I was just using.       I never know if something is under Features, Network, Player, or System.   They all seem so similar in purpose.   Would be great to combine those into a single screen with a sub-selector, or something...   It's just so tedious to fat thumb hit X, fat thumb hit settings, fat thumb hit oops... missed... fat thumb hit X then settings, fat thumb hit system - oops wrong menu, where was that thing?   man i'm getting tired...

**Comment by subatomicglue** [comment]
Big Thumb, Little UI:
![IMG_3415](https://user-images.githubusercontent.com/7660853/141404826-e0645540-26d4-4e5f-94f2-7c59e8efc3b9.jpg)
Little UI:
![IMG_3413](https://user-images.githubusercontent.com/7660853/141404819-b18544bf-9777-4c59-a13e-10f152d7a042.jpg)

**Comment by R101G** [comment]
@subatomicglue  I have the same problem, but only on the drop-down menus. One thing I find helps is having touch feedback enabled in the tablet's developer mode menu - at least it shows how far out my aim is.

**Comment by rern** [comment]
There's a scrollbar on the right which should means it's a local display.
So: Settings > Features > Browser on RPi > Setting - Zoom

@R101-Github,
Let's see it on your tablet screen.

**Comment by R101G** [comment]
@rern 
Thanks, but it's perfectly useable as is. Anyone with better tactile skills or smaller fingers would not have a problem and if I am making a lot of changes I can use a PC and mouse. Perhaps making the drop-down menus a bit bigger would be nice when you have the time.

**Comment by subatomicglue** [comment]
200% zoom, and lost the top and bottom bars, no settings button anymore

![IMG_3423](https://user-images.githubusercontent.com/7660853/141492322-68e2094c-3510-4422-93be-9024932cf04d.jpg)

**Comment by subatomicglue** [comment, marked as answer]
# Solved
Zoom == 180 did the trick.   (190 still had the problem).
Was able to connect with the phone after rebooting....  (I have a mesh network (eero 6 pro) and sometimes the wifi connection gets lost on the rAudio-1...  not sure why.   but i suspect that the rAudio doesn't roam to the strongest signal (I have like 12 'elephant' wifi names all the same which is my access point SSD name...).

Thanks for the tip - was unaware of "Browser on RPi" being useful for display size,  though I had scrolled through that list a dozen times.  I think the list item just didn't register to me as affecting "display size"

![IMG_3424](https://user-images.githubusercontent.com/7660853/141494116-b2a0b748-7aa1-4c49-b96a-57aaba6350ba.jpg)

My thumb in relation to the menu - a lot better now at 180 zoom:
![IMG_3425](https://user-images.githubusercontent.com/7660853/141497179-6e8cd081-cc45-4691-9e6e-6c2978f317b3.jpg)

I can now read the menu option that tells me about "display settings",    (well, actually "Browser on RPi"...  cryptic name to a user not totlaly familiar with the tech behind this UI/Ux here, yes?):
![IMG_3426](https://user-images.githubusercontent.com/7660853/141497427-8a3e6f48-d1b0-4638-9351-9e4a865cf97f.jpg)
